backend/src/routers/user_auth/utils.py

```diff
@@ -1,46 +1,51 @@
+import logging
+from getpass import getpass
 from datetime import datetime, timedelta
-from fastapi import HTTPException
 from fastapi.security import OAuth2PasswordBearer
 from passlib.context import CryptContext
-from pydantic import BaseModel  # pylint: disable=E0611
 import jwt
-from starlette.status import HTTP_403_FORBIDDEN
+from mongoengine import (
+    connect,
+    StringField,
+    BooleanField,
+    Document,
+    DoesNotExist,
+    MultipleObjectsReturned,
+)
 
-# to get a string like this run:
-# openssl rand -hex 32
-
-# Secret key will be moved from here, development purposes only.
 SECRET_KEY = "09d25e094faa6ca2556c818166b7a9563b93f7099f6f0f4caa6cf63b88e8d3e7"
 
 ALGORITHM = "HS256"
 REFRESH_TOKEN_EXPIRE_MINUTES = 360
-ACCES_TOKEN_EXPIRES_MINUTES = 15
-
-MOCK_DB = {
-    "johndoe": {
-        "username": "ada",
-        "full_name": "John Doe",
-        "email": "johndoe@example.com",
-        "hashed_password": "$2b$12$EixZaYVK1fsbw1ZfbX3OX\
-ePaWxn96p36WQoeG6Lruj3vjPGga31lW",
-        "disabled": False,
-    }
-}
+ACCESS_TOKEN_EXPIRES_MINUTES = 15
 
 PWD_CONTEXT = CryptContext(schemes=["bcrypt"], deprecated="auto")
 
 OAUTH2_SCHEME = OAuth2PasswordBearer(tokenUrl="/login")
 
 
-class User(BaseModel):  # pylint: disable=R0903
-    username: str
-    email: str = None
-    full_name: str = None
-    disabled: bool = None
+class User(Document):
+    username = StringField(required=True, max_length=200)
+    email = StringField(required=True, max_length=200)
+    hashed_password = StringField(required=True, max_length=200)
+    disabled = BooleanField(required=True)
```

Comment:
It shouldn't be required to set `disabled`, make it optional but set default value to `False`